internal/clients/sources/stubs/sources_stub.go

```diff
@@ -0,0 +1,56 @@
+package stubs
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients/sources"
+)
+
+type sourcesCtxKeyType string
+
+var sourcesCtxKey sourcesCtxKeyType = "sources-interface"
+
+type SourcesIntegrationStub struct {
+	store *[]sources.Source
+}
+
+func init() {
+	sources.GetSourcesClient = getSourcesClient
```

Comment:
This could have been simplified :) I use one more method, because I'm calling the method from two different places with differet params, but we have just one caller with fixed params here :)
```suggestion
	sources.GetSourcesClient = getSourcesClientStub
```

Reply:
Do you mean like that? :) 
```golang
func init() {
	sources.GetSourcesClient = getSourcesClientStub
}

func getSourcesClientStub(ctx context.Context) (si sources.SourcesIntegration, err error) {
	var ok bool
	if si, ok = ctx.Value(sourcesCtxKey).(*SourcesIntegrationStub); !ok {
		err = &contextReadError{}
	}
	return si, err
}
```

Reply:
> Two general not important comments:
> 
> * I'd not necessarily expect this to be as dynamic as our  database stubs, so I'd be ok with making the data static, but this is also fine, it might be useful
> * I'd like to verify the account header being sent as well, but not necessarily in this PR and not necessarily as part of these tests (I don't know what would be the best)
> 
> So I'm happy with what we have here :)

I thought of using the fixtures idea like we used to but I wasn't sure what is the right approach, therefore I am asking now :)

Reply:
I've not been sure how to fixture neighter, so I'd just not for now and keep the approach you used.
What I meant was having static data in thu stub directly, but this is bit more flexible, so lets just keep what you have :)